File: src/main/java/com/gmail/borlandlp/minigamesdtools/gui/inventory/slots/InventoryGuiSlotCreatorHub.java
package com.gmail.borlandlp.minigamesdtools.gui.inventory.slots;

import com.gmail.borlandlp.minigamesdtools.creator.AbstractDataProvider;
import com.gmail.borlandlp.minigamesdtools.creator.CreatorHub;

public class InventoryGuiSlotCreatorHub extends CreatorHub {
    public InventorySlot createSlot(String ID, AbstractDataProvider dataProvider) throws Exception {
        return (InventorySlot) this.create(ID, dataProvider);
    }
}

File: src/main/kotlin/com/gmail/borlandlp/minigamesdtools/gui/inventory/slots/InventoryGuiSlotCreatorHub.kt
package com.gmail.borlandlp.minigamesdtools.gui.inventory.slots

import com.gmail.borlandlp.minigamesdtools.creator.AbstractDataProvider
import com.gmail.borlandlp.minigamesdtools.creator.CreatorHub

class InventoryGuiSlotCreatorHub : CreatorHub() {
    @Throws(Exception::class)
    override fun create(itemID: String, dataProvider: AbstractDataProvider): InventorySlot {
        return super.create(itemID, dataProvider) as InventorySlot
    }
}